daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala

```diff
@@ -1634,7 +1659,7 @@ class EngineTest
       err.msg should include("precondition violation")
     }
 
-    "not be create if has an empty set of maintainer" in {
+    "crash if uses a key with an empty set of maintainers" in {
```

Comment:
This is still a create no? Seems a bit confusing to rename this to basically the same name as the test above when one is about create and the other is about lookup.

Reply:
reverted.